doc/herbstluftwm.txt

```diff
@@ -1415,11 +1418,17 @@ listed as follows:
 |s - title                | its window title
 |s - keymask              | [[KEYMASK]]A regular expression that is matched
                             against the string representation of all key
-                            bindings (see list_keybinds). While this client is
-                            focused, only bindings that match the expression
-                            will be active. Any other bindings will be disabled.
-                            The default keymask is an empty string (""), which
-                            does not disable any keybinding.
+                            bindings (as they are printed by list_keybinds).
+                            While this client is focused, only bindings that
+                            match the expression will be active. Any other
+                            bindings will be disabled. The default keymask is an
+                            empty string (""), which does not disable any
+                            keybinding.
+|s - keys_inactive        | A regular expression that describes which
+                            keybindings are inactive for the client. If a key
```

Comment:
"for the client" → "while the client is focused"?

Reply:
Yes, thanks!